internal/cmdrun/organization/list/list.go

```diff
@@ -0,0 +1,57 @@
+package list
+
+import (
+	"context"
+	"strconv"
+	"vf-admin/internal/utils"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+// CmdRunE is what's executed when running `vf-admin organization list`
+func CmdRunE(cmd *cobra.Command, args []string) error {
+	// Create the API client using the authentication key for requests
```

Comment:
Prob copied my code - should be `// Create the API client`, ignore the auth key part